src/app/user/resource.ts

```diff
@@ -25,12 +25,12 @@ export const create = async (payload: any) => {
 export const getListWithFilters = async (query: JSObj) => {
   // ? 1) Get all users with all policies
 
-  const roleTagFilter = extractRoleTagFilter(query);
+  const roleTagFilter = extractRoleTagFilter(query) || [];
```

Comment:
In case of multi tags filter, how will the queryparams look like? And will it be parsed to array?

Reply:
We are using `qs` as [query parser]( https://github.com/odpf/shield/blob/main/src/config/composer.ts#L23)
it supports 4 options to pass array

```
qs.stringify({ a: ['b', 'c'] }, { arrayFormat: 'indices' })
// 'a[0]=b&a[1]=c'
qs.stringify({ a: ['b', 'c'] }, { arrayFormat: 'brackets' })
// 'a[]=b&a[]=c'
qs.stringify({ a: ['b', 'c'] }, { arrayFormat: 'repeat' })
// 'a=b&a=c'
qs.stringify({ a: ['b', 'c'] }, { arrayFormat: 'comma' })
// 'a=b,c'

```